File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.js
function solution(my_string) {
    var answer = [];

    for (let i = 0; i < my_string.length; i++) {
        let ch = my_string[i];
        if (!isNaN(ch) && ch !== ' ') {
            answer.push(Number(ch));
        }
    }
    answer.sort((a, b) => a - b);
    return answer;
}

File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.java
import java.util.Arrays;

class Solution {
    public int[] solution(String my_string) {
        int count = 0;
        
        for (int i = 0; i < my_string.length(); i++) {
            char c = my_string.charAt(i);
            if (c >= '0' && c <= '9') {
                count++;
            }
        }
        
        int[] answer = new int[count];
        
        int index = 0;
        for (int i = 0; i < my_string.length(); i++) {
            char c = my_string.charAt(i);
            if (c >= '0' && c <= '9') {
                answer[index] = c - '0';
                index++;
            }
        }
        
        Arrays.sort(answer);
        return answer;
    }
}